src/Data/Metadata.cs

```diff
@@ -140,6 +141,17 @@ public class KernelProperties
             get {
                 yield return (KernelName, KernelVersion);
                 yield return ("Jupyter Core", typeof(KernelProperties).Assembly.GetName().Version.ToString());
+                yield return (
+                    ".NET Runtime",
+                    // Use the technique documented at
+                    //     https://weblog.west-wind.com/posts/2018/Apr/12/Getting-the-NET-Core-Runtime-Version-in-a-Running-Application
+                    // to get the target framework moniker of the entry point for the kernel,
+                    // so that we can report that as a version.
+                    Assembly
+                    .GetEntryAssembly()
+                    ?.GetCustomAttribute<TargetFrameworkAttribute>()
+                    ?.FrameworkName
```

Comment:
```suggestion
                    ?.FrameworkName ?? ""
```
I think this is safer; this way you'll get an empty string instead of a null string pointer.

Reply:
Thanks for catching that, I'll go on and do that, then.